packages/react/src/components/browserPreview/BrowserPreview.tsx

```diff
@@ -6,7 +6,16 @@ import {Svg} from '../svg';
 import {Tooltip} from '../tooltip';
 
 export interface BrowserPreviewProps {
+    /**
+     * Text that will be displayed in the tooltip on the i icon
+     *
+     * @default The final look in your search page may differ due to the customization you made in your page.
```

Comment:
That's the default value?

Reply:
Yes :smile: 
![image](https://user-images.githubusercontent.com/63734941/158379818-fe524bc7-fd99-401e-9f4c-f08e36aa2eda.png)